partner_private_information/views/res_partner_view.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="res_partner_form_inherit_private_info">
+        <field name="name">res.partner.form.inherit.private.info</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//notebook" position="inside">
+                <page
+                    string="Private Info"
+                    name="private_info"
+                    groups="partner_private_information.group_partner_private_info"
+                >
+                    <group>
+                        <group id="left_group">
+                            <span class="o_form_label o_td_label" name="address_name">
+                                <field
+                                    name="type"
+                                    invisible="is_company"
+                                    readonly="user_ids"
+                                    required="not is_company"
+                                    class="fw-bold"
+                                />
+                                <b invisible="not is_company">Address</b>
+                            </span>
+                            <div class="o_address_format">
+                                <field
+                                    name="private_street"
+                                    placeholder="Street..."
+                                    class="o_address_street"
+                                    readonly="type == 'contact' and parent_id"
```

Comment:
Note that all the fields with "readonly="type == 'contact' and parent_id" will never be populated. I think you should take a look at _children_sync function for res.partner in base module.

Reply:
Applied _children_sync function. Can you check it?